models/zeroex/ethereum/zeroex_ethereum_api_fills.sql

```diff
@@ -86,25 +85,25 @@ v3_fills_no_bridge AS (
             evt_block_time                                                             AS block_time,
             fills.makerAddress                                                         AS maker,
             fills.takerAddress                                                         AS taker,
-            '0x' || SUBSTRING(fills.takerAssetData, 35, 40)                            AS taker_token,
-            '0x' || SUBSTRING(fills.makerAssetData, 35, 40)                            AS maker_token,
-            fills.takerAssetFilledAmount                                               AS taker_token_amount_raw,
-            fills.makerAssetFilledAmount                                               AS maker_token_amount_raw,
+            bytearray_substring(fills.takerAssetData, 34, 40) AS taker_token,
+            bytearray_substring(fills.makerAssetData, 34, 40) AS maker_token,
+            cast(fills.takerAssetFilledAmount  as int256) AS taker_token_amount_raw,
+            cast(fills.makerAssetFilledAmount  as int256)AS maker_token_amount_raw,
```

Comment:
we've been writing `amount_raw` fields as `uint256` in the entire `dex.trades` lineage, which this eventually feeds into. can you universally update that in this spell?